bin/include/functions.sh

```diff
@@ -27,6 +27,28 @@
 #   source "${IGNITE_HOME_TMP}"/bin/include/functions.sh
 #
 
+# Extract java version to `version` variable.
+javaVersion() {
+    version=$("$1" -version 2>&1 | awk -F '"' '/version/ {print $2}')
+}
+
+# Extract only major version of java to `version` variable.
+javaMajorVersion() {
+    javaVersion "$1"
+    version="${version%%.*}"
+
+    if [ ${version} -eq 1 ]; then
+        # Version seems starts from 1, we need second number.
+        javaVersion "$1"
+        backIFS=$IFS
```

Comment:
AFAIK, there is no need in saving file separator if IFS is redefined in the same string with the command itself without semicolon

Reply:
If you're talking about backIFS, yes I need to restore it, because if not to do this, it breaks our scripts.